Xamarin.Forms.Core/MergedStyle.cs

```diff
@@ -5,163 +5,160 @@
 
 namespace Xamarin.Forms
 {
-	public partial class VisualElement
+	sealed class MergedStyle : IStyle
```

Comment:
I didn't actually change the formatted here, GitHub just thinks I did. All that happened here was the public partial class VisualElement was removed to allow the MergedStyle class to be used by a Span.